embeddings/bq-vector-search-log-outlier-detection.ipynb

```diff
@@ -0,0 +1,1474 @@
+{
```

Comment:
<p>Can you add a link to Cloud Audit Logs <a href="https://cloud.google.com/logging/docs/audit" target="_blank">https://cloud.google.com/logging/docs/audit</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/381/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>